StateWrappers.as

```diff
@@ -18,6 +18,13 @@ namespace VehicleState
 		RouletteSuper,
 		RouletteUltra
 	}
+
+	shared enum VehicleType {
+		Stadium,
+		Snow,
+		// Rally,  // to update when car is added
+		// Desert  // to update when car is added
+	}
```

Comment:
Is this enum not defined by the game itself? It's returned from `GetVehicleType` through a value in the vis state.

Reply:
I don't understand, I'm not aware of any enum defined by the game for vehicle type

Reply:
Alright, I don't see an obvious one either. However, I think we do need to stay consistent and give these enum values the right names: `CarStadium`, `CarSnow`.

Reply:
Well the stadium car is called CarSport, but I can change it to that if you feel that's better